2.4/examples/replica/README.md

```diff
@@ -25,38 +24,39 @@ OpenShift Origin v3.
 This resource defines a headless Kubernetes Service that serve as an entrypoint
 to the MongoDB cluster. The Service endpoints are the Pods created by the
 MongoDB replication controller.
+
 Having 'headless' Service means that the `portalIP` attribute of this Service is
-set to `None`. This allows to use DNS queries to discover endpoints of the
-service from inside the container, by quering the Service name (eg. `dig mongodb
-A +short +search`).
+set to `None`. This allows to use DNS queries to discover other MongoDB
+endpoints from inside the container, by quering the Service name (eg. `dig
+mongodb A +short +search`).
 
 ### Pod 'mongodb-service'
 
-This Pod serves as an replSet initialization member. Inside the
-'initiate_replica.sh' command is ran, which will wait for all members created by
-the ReplicationController to become available and then creates the initial
-MongoDB replica set. This set is then distributed to all members.
-This Pod is a 'run-once' Pod, which means that once the replSet is initialized
-that the members receive the replication data, this Pod is destroyed.
+This pod is responsible for initializing the MongoDB replica set. It runs just
+once when the resources are being created for the first time. It also creates
+the initial database and users. After the initialization is completed and data
+are replicated to other member of MongoDB replication set, this pod gave up the
+PRIMARY role in the replication set and shutdown.
```

Comment:
most of the time we are using the `replications` but on line 35 we use `replica`. Would be nice to unify it.